README.md

```diff
@@ -37,7 +38,15 @@ There are two options:
 Alternately, run `gcloud auth application-default login --impersonate-service-account data-download@hastings-picarro.iam.gserviceaccount.com` and follow the steps to log in.
 
 ## Downloading data
-Once you've installed the package and logged in, you can use the provided scripts to download data from the cloud.
+Once you've installed the package and logged in, you can use the provided scripts to download data from the cloud. Once you've set up the first time, you will no longer need to do the steps above (logging in to google cloud, installing via pip). However, you will need to navigate to the directory you used before and reactivate your virtual environment via:
+```
+source venv/bin/activate
+```
+on Mac or 
+```
+source .venv\Scripts\activate.bat
+```
+on Windows.
```

Comment:
This actually depends on which terminal they are using. I think we can add that for windows, we assume that you are using `powershell` at the very beginning